### PR TITLE
Silicons are dusted by instant kill rifles

### PR DESCRIPTION
**Pull request by coiax**
Why should carbons get all the fun?

### CODE REVIEW

**Review by alexkar598 — COMMENTED** [review]
What's the change in line 147?

**Comment by coiax** [comment]
The file originally didn't have a newline at the end of file.

**Comment by Cyberboss** [comment]
Silicons can be gibbed fyi

**Comment by coiax** [comment]
@Cyberboss I think gibbing silicons leaves the MMI behind though. Maybe?

**Comment by Cyberboss** [comment]
Use the nobrain param

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -144,4 +144,8 @@
 	if(iscarbon(target))
 		var/mob/living/carbon/M = target
 		M.visible_message("<span class='danger'>[M] explodes into a shower of gibs!</span>")
-		M.gib()
\ No newline at end of file
+		M.gib()
+	if(issilicon(target))
+		var/mob/living/silicon/S = target
```
Comment (RemieRichards) [inline code/modules/projectiles/projectile/beams.dm]: you don't use this.

File: code/modules/projectiles/projectile/beams.dm [hunk 2]
```diff
@@ -144,4 +144,8 @@
 	if(iscarbon(target))
 		var/mob/living/carbon/M = target
 		M.visible_message("<span class='danger'>[M] explodes into a shower of gibs!</span>")
-		M.gib()
\ No newline at end of file
+		M.gib()
+	if(issilicon(target))
+		var/mob/living/silicon/S = target
+		S.visible_message("<span class='danger'>[M] turns to dust!</span>")
```
Comment (PKPenguin321) [inline code/modules/projectiles/projectile/beams.dm]: missed another one

**Comment by KorPhaeron** [comment]
stale

**Comment by PKPenguin321** [comment]
It's a one character fix just use your maintainer voodoo to edit it and merge

**Pull request closed without merging**